oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -340,7 +342,11 @@ private void registerNodeStore() throws IOException {
             String db = config.db();
             boolean soKeepAlive = config.socketKeepAlive();
 
-            MongoClientURI mongoURI = new MongoClientURI(uri);
+            ConnectionString mongoURI = new ConnectionString(uri);
+            MongoClientSettings settings = MongoClientSettings.builder()
+                    .applyConnectionString(mongoURI)
```

Comment:
variable `mongoURI` can be inlined.
```suggestion
                    .applyConnectionString(new ConnectionString(uri))
```

Reply:
Done

Reply:
I had to revert this because the `mongoURI` is [used a few lines below](https://github.com/apache/jackrabbit-oak/pull/2226/files/974372ee2b6d26cfb898864d58eaa45b0a352a7b#diff-0e315af2cc5cc1327210891e5ba43454d739fd8bcdcfefb016d985f071a35cb2R359).